yamllint/rules/document_end.py

```diff
@@ -99,11 +99,13 @@ def check(conf, token, prev, next, nextnext, context):
         prev_is_end_or_stream_start = isinstance(
             prev, (yaml.DocumentEndToken, yaml.StreamStartToken)
         )
+        prev_is_directive = isinstance(prev, (yaml.DirectiveToken))
```

Comment:
I allowed myself to update the pull request with a more Pythonic version:
```suggestion
        prev_is_directive = isinstance(prev, yaml.DirectiveToken)
```